docs/helm-chart/index.rst

```diff
@@ -105,6 +109,13 @@ To upgrade the chart with the release name ``airflow``:
 
     helm upgrade airflow apache-airflow/airflow --namespace airflow
 
+**Explanation**
+- `helm upgrade` is a command to upgrade a release
+- the first 'airflow' argument is the release name
+- 'apache-airflow/airflow' is the Helm Chart
+- the second 'airflow' argument is the namespace
+
+
```

Comment:
similar to above, i would suggest that you amend existing wording if not clear already rather than adding an explanation section.

I would _not_ document the argument syntax --- that's not our role in airflow, that is the job of the helm docs.